src/main/java/org/broadinstitute/hellbender/utils/nio/SeekableByteChannelPrefetcher.java

```diff
@@ -441,6 +447,13 @@ public boolean isOpen() {
     @Override
     public void close() throws IOException {
         if (open) {
+            exec.shutdown();
+            try {
+                exec.awaitTermination(3, TimeUnit.MINUTES);
```

Comment:
A zero timeout wouldn't kill the thread, it would just stop waiting for it to acknowledge the interruption. Then the thread would be reading on a closed channel and would fail. This isn't a huge deal it would just confuse matters.

How about a much shorter timeout? I expect that five seconds is still enough for most cases.